Minor fixes to the SyntaxError: missing variable name page (#14863)
mdn/content: files/en-us/web/javascript/reference/errors/no_variable_name/index.md

A variable is missing a name. The cause is most likely a typo or a forgotten variable name.
Make sure that you've provided the name of the variable before the `=` sign.

When declaring multiple variables at the same time, make sure that the previous lines/declaration does not end with a comma instead of a semi-colon.

## Examples

### Missing a variable name

```js example-bad
const = "foo";
```

> Coming up with a descriptive variable name is challenging for most developers.
> It gets easier over time.

Reviewer: ```suggestion
It is easy to forget to assign a name for your variable!
```

comma? Did you accidentally add commas instead of semicolons?

```js example-bad

> const x, y = "foo",

Reviewer: See comment below. This won't work "as an example" if you use const because it will be invalid whether you use semicolons or commas. If you want to make this something other than `var` then perhaps "let".

Author: I think I fixed it.

const x, = "foo"

const first = document.getElementById('one'),
const second = document.getElementById('two'),

// SyntaxError: missing variable name
```

The fixed version:

```js example-good

> const x, y = "foo";

Reviewer: The original `var x, y = "foo";` was wrong. Essentially it results in x being unassigned. You can do that with a var but not a const, so this will fail.

Author: I think I fixed it.
